### PR TITLE
Update to 2021.2 EAP

### PR DESCRIPTION
**Pull request by ThexXTURBOXx**
Basic functionality seems to work :)

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -15,8 +15,8 @@ repositories {
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-    version = "2021.1"
-    setPlugins("java", "Kotlin")
+    version.set("212-EAP-SNAPSHOT")
+    plugins.set("java, Kotlin".split(',').map(String::trim))
```
Comment (Juuxel) [inline build.gradle.kts]:
```suggestion
    plugins.set(listOf("java", "Kotlin"))
```

File: build.gradle.kts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    id("org.jetbrains.intellij") version "0.7.2"
-    kotlin("jvm") version "1.4.32" // Corresponds to IDEA 2021.1, see kt jars inside the ideaIC dep in IDEA
-    id("org.jmailen.kotlinter") version "3.4.0"
+    id("org.jetbrains.intellij") version "1.0"
+    kotlin("jvm") version "1.4.32" // Corresponds to IDEA 2021.2 EAP, see kt jars inside the ideaIC dep in IDEA
```
Comment (Juuxel) [inline build.gradle.kts:5]: Just making sure, did you check this? They might have updated to Kotlin 1.5.

**Review by Juuxel — CHANGES_REQUESTED** [review]
Thanks for this PR! I can merge it once 2021.2 gets a stable release, since maintaining two separate branches needs some more work.

File: build.gradle.kts (hunk 2 above)
Comment (ThexXTURBOXx) [reply build.gradle.kts:5]: I checked that and it seems like they are using 1.4 still. I will, however, check again!

Comment (ThexXTURBOXx) [reply build.gradle.kts:5]: Checked again, they are still using 1.4.32 for now. If they update it in another EAP version (I don't believe they would...), I will update my PR accordingly

**Comment by ThexXTURBOXx** [comment]
> 
> 
> Thanks for this PR! I can merge it once 2021.2 gets a stable release, since maintaining two separate branches needs some more work.

Yes, no problem! Does your license allow me to upload a 2021.2 EAP version of your plugin to my webspace and link to it here for other users?

**Comment by Juuxel** [comment]
It does, as long as you follow its other terms. Also for now, I think I can somehow update the version range so that the current version built on 2021.1 also works on 2021.2 EAP.

**Comment by ThexXTURBOXx** [comment]
Okay perfect. Then I will read through them later, thanks!
I could also adjust this PR such that 2021.2 EAP is supported next to 2021.1, if you want.

**Comment by ThexXTURBOXx** [comment]
Now the plugin should support both at the same time

**Review by Juuxel — APPROVED** [review]

**Comment by Juuxel** [comment]
Thanks for the update! I'll probably change this locally to build on 2021.1 still, but future releases should support 2021.2 too.

**Pull request merged**